main.js

```diff
@@ -0,0 +1,35 @@
+var swiper = new Swiper('.swiper--preview-img', {
```

Comment:
- Ya los var no se usan. Revisa donde lo usas y actualiza a `let` o `const`. Si tienes dudas sobre su uso, puedes preguntarme.
- Esto es nuevo. Cuando vas a interactuar con un elemento del dom desde javascript una buena practica es colocarle una clase adicional al elemento que comience con `js-`. Con esto se sabe que hay un js que interactua con este elemento
```suggestion
const swiper = new Swiper('.js-swiper-preview-img', {
```